example/feature_demo/demo_service.proto

```diff
@@ -38,25 +49,38 @@ message UpdateIntPointResponse {
 }
 
 message DeleteIntPointRequest {
-        uint32 id = 1;
+    // Only the id is needed for a delete request
+    uint32 id = 1;
 }
 
+// By convention, on DELETE no response is expected
+
 message ListIntPointResponse {
+    // Note repeated field and plural name 'results'
     repeated int_point results = 1;
 }
 
+// A dummy type to demo an rpc that can't be autogenerated
 message Something {
 }
 
 service IntPoint {
+  // This option tells protoc-gen-gorm to generate the calls and stubs
   option (gorm.server).autogen = true;
+  // The convention names are Create/Read/Update/List/Delete. Since no type is
+  // specified, only one set of CURDL operations can be defined in each service
```

Comment:
is this true? the type is inferred from the request/response wrapper, so it seems like you could have a `CreateFoo` and `CreateBar` and both could get generated.